### ISSUE TITLE
When new branch for issue created add the user as assignee

### ISSUE THREAD

**Issue by vrom911** [kowainik/hit-on#15]
Blocked by #108 

If there are rights for that.

**Comment by StevenXL** [comment]
@vrom911 I'd like to take a stab at this issue. Can you assign it to me. Thank you!

**Comment by vrom911** [comment]
Yes, sure!
After thinking a while on this issue I guess it's better to make this as the option like `hit new 10 --assign` (`--assign`/`-a`), so it won't be annoying to anyone who is not willing to be assign :)

**Comment by StevenXL** [comment]
That makes sense! Thank you.

**Comment by StevenXL** [comment]
Hm.. I've cloned the project, but when I run `stack ghci`, I run into the following error:

```
Using main module: 1. Package `hit-on' component exe:hit with main-is file: /Users/stevenleiva/Repos/hit-on/app/Main.hs
Building all executables for `hit-on' once. After a successful build of all of them, only specified executables will be rebuilt.
hit-on-0.0.0: initial-build-steps (lib + exe)
The following GHC options are incompatible with GHCi and have not been passed to it: -threaded
Configuring GHCi with the following packages: hit-on
GHCi, version 8.6.3: http://www.haskell.org/ghc/  :? for help

<interactive>:1:1: error:
    attempting to use module ‘main:Prelude’ (/Users/stevenleiva/Repos/hit-on/src/Prelude.hs) which is not loaded
```

Can I get some pointers on how to fix this?

**Comment by vrom911** [comment]
Hmm. I don't really now, how I can help with this. I just build with stack and used `ghci` and I had the same issue. I can even see some tickets in `Stack` about the problem.  Meanwhile `cabal new-repl` works without any problems so I would recommend you to use that, because it looks like `Stack` specific problem, not the project problem, sorry.

**Comment by StevenXL** [comment]
@vrom911 No problem. When I run `stack build` it works, so I'll just use `stack build --fast --file-watch` to make sure the changes result in a well-typed program.

**Comment by StevenXL** [comment]
@vrom911 Implemented vi PR #59. Review welcomed.
